crates/sui-bridge/src/server/handler.rs

```diff
@@ -165,7 +165,8 @@ where
             Err(e) => {
                 match e {
                     // Only cache non-transient errors
-                    BridgeError::BridgeEventInUnrecognizedSuiPackage
+                    BridgeError::GovernanceActionIsNotApproved { .. }
```

Comment:
Seems fragile when adding new error types. Somewhere you would need a full match

Reply:
I tried to add a full match but it turns out to be very awkward and tedious. In the context of serving handling a request, the possible non-transient error list is limited, so i think we are fine with this for now